pdc/settings_test.py

```diff
@@ -48,3 +48,6 @@
 }
 
 COMPONENT_BRANCH_NAME_BLACKLIST_REGEX = r'^epel\d+$'
+
+# Set the key_id of sigkey "redhatrelease2" as default
```

Comment:
The comment should say `name` of key, not `key_id`.

Reply:
Yes, will change it.